.gitignore

```diff
@@ -22,3 +22,5 @@ test/native/junit.xml
 
 # Local overrides
 .wp-env.override.json
+
+.idea
```

Comment:
a minor one: is this something that we want to add here?

Reply:
it's because IntelliJ's WebStorm keeps config settings in there. We removed it